### PR TITLE
Hotfix/deploy url trying to fix js routes

### PR DESCRIPTION
**Pull request by nmpribeiro**
Fixes GH Pages backwards routing by switching `BrowserRouter` for `HashRouter`

### CODE REVIEW

**Comment by nmpribeiro** [comment]
@VioletteHeron check this solution (for now and for GH Pages)

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "undp-radar-app",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://sdg-ai-lab.github.io/Digital_Technologies_Radar",
+  "homepage": "/Digital_Technologies_Radar",
```
Comment (nmpribeiro) [inline package.json:5]: Better be generic as possible

Comment (OzgeOzkaya) [reply package.json:5]: got it now. @VioletteHeron this part changed, just a reminder :)

File: src/App.tsx [hunk 2]
```diff
@@ -11,12 +11,16 @@ export const App: React.FC = () => {
...
-        {/*Note : this line is a quickfix to have the radar display
-        on the homepage with the first iteration of deployment through gh-pages. */}
-        <BrowserRouter basename='/Digital_Technologies_Radar'>
+        {/* While we are in GH Pages we will need to use this HashRouter
+         * source: https://www.freecodecamp.org/news/deploy-a-react-app-to-github-pages/
+         *
+         * When we move to a better solution, bring back the <BrowserRouter />
+         * and replace HashRouter
+         **/}
+        <HashRouter>
```
Comment (nmpribeiro) [inline src/App.tsx:20]: @VioletteHeron do not forget to see this

File: src/App.tsx [hunk 3]
```diff
@@ -11,12 +11,16 @@ export const App: React.FC = () => {
...
           <NavApp />
           <BlipView />
-        </BrowserRouter>
+        </HashRouter>
```
Comment (nmpribeiro) [inline src/App.tsx:23]: @VioletteHeron do not forget to see this

File: src/App.tsx [hunk 4]
```diff
@@ -11,12 +11,16 @@ export const App: React.FC = () => {
   return (
     <AppRadarProvider>
       <ChakraUI>
-        {/*Note : this line is a quickfix to have the radar display
-        on the homepage with the first iteration of deployment through gh-pages. */}
-        <BrowserRouter basename='/Digital_Technologies_Radar'>
+        {/* While we are in GH Pages we will need to use this HashRouter
```
Comment (nmpribeiro) [inline src/App.tsx:14]: @VioletteHeron do not forget to see this

**Pull request merged**